### PR TITLE
Fix month names + Add Firefox in karma

### PR DESCRIPTION
**Pull request by adrienDog**

### CODE REVIEW

**Comment by adrienDog** [comment]
To fix issue #11 @StevePole

File: angular/components/tw-date/tw-date.controller.js [hunk 1]
```diff
@@ -314,7 +310,7 @@
 		}
 
 		function getIsoDateWithoutTime(dateAsISOString) {
-			return dateAsISOString.substr(0, dateAsISOString.indexOf('T'));
+			return dateAsISOString.substring(0, dateAsISOString.indexOf('T'));
```
Comment (StevePole) [inline angular/components/tw-date/tw-date.controller.js]: Why this change?

File: angular/components/tw-date/tw-date.directive.spec.js [hunk 2]
```diff
@@ -473,6 +473,7 @@ describe('Directive: TwDate', function() {
             });
         });
         describe('with month select', function() {
+            // TODO: Fix for Firefox
```
Comment (StevePole) [inline angular/components/tw-date/tw-date.directive.spec.js]: TODO?

File: angular/components/tw-date/tw-date.controller.js (hunk 1 above)
Comment (adrienDog) [inline angular/components/tw-date/tw-date.controller.js]: just for consistency with the other usages and also because I felt the definition of `substring` was more similar to what we want here (even though in practice it is the same effect here)
http://stackoverflow.com/questions/3745515/what-is-the-difference-between-substr-and-substring

File: angular/components/tw-date/tw-date.directive.spec.js (hunk 2 above)
Comment (adrienDog) [inline angular/components/tw-date/tw-date.directive.spec.js]: It appeared when I added FF as a browser in karma: it works in real FF, but in the tests somehow I think the triggerHandler doesn't work.
I'll take a quick look sry didnt think it was urgent to tackle

Comment (StevePole) [inline angular/components/tw-date/tw-date.directive.spec.js]: It might not be urgent, I just saw "Fix for Firefox" and thought, surely we want to support firefox...  Perhaps explain that it does not affect the real browser.

Comment (adrienDog) [inline angular/components/tw-date/tw-date.directive.spec.js]: It does sound scary when you read it ^^

**Comment by StevePole** [comment]
:+1:
